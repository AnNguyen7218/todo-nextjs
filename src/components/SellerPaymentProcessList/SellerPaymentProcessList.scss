@import 'config.gutters';
@import 'config.media';

.container {
  margin-bottom: gutter(24);

  @include break(xl) {
    margin-top: gutter(24);
    margin-bottom: gutter(96);
  }
}

.title {
  margin-bottom: gutter(32);

  @include break(xl) {
    margin-bottom: gutter(48);
  }
}

.process-container {
  display: flex;
  flex-direction: column;

  @include break(md) {
    flex-direction: row;
    justify-content: space-between;
  }
}
