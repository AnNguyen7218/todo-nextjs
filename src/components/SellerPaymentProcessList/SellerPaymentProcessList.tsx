import React from 'react';
import { Container, SellerPaymentProcess } from 'components';
import { SELLER_PAYMENT_PROCESSES } from 'constants/sellerLandingPage';
import css from './SellerPaymentProcessList.scss';

const SellerPaymentProcessList = () => {
  return (
    <Container className={css['container']}>
      <h2 className={css['title']}>Getting paid is now simple and secure</h2>
      <div className={css['process-container']}>
        {SELLER_PAYMENT_PROCESSES.map((process, index) => {
          return <SellerPaymentProcess process={process} key={index} />;
        })}
      </div>
    </Container>
  );
};

export default SellerPaymentProcessList;
