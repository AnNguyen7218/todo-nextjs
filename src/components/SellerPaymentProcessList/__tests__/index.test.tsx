import React from 'react';
import { render, cleanup } from '@testing-library/react';
import SellerPaymentProcessList from '../SellerPaymentProcessList';

afterEach(cleanup);

describe('SellerPaymentProcess Component', () => {
  test('SellerPaymentProcess should match snapshot', () => {
    const { container } = render(<SellerPaymentProcessList />);
    expect(container).toMatchSnapshot();
  });
});
