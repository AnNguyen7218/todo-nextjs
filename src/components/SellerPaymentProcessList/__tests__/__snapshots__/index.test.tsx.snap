// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellerPaymentProcess Component SellerPaymentProcess should match snapshot 1`] = `
<div>
  <div
    class="container container"
  >
    <h2
      class="title"
    >
      Getting paid is now simple and secure
    </h2>
    <div
      class="process-container"
    >
      <div
        class="container"
      >
        <ic-mail.svg
          classname="logo"
        />
        <h3
          class="title"
        >
          Pick your price
        </h3>
        <p>
          Take charge of your earnings. With us, you always control your prices.
        </p>
      </div>
      <div
        class="container"
      >
        <ic-language.svg
          classname="logo"
        />
        <h3
          class="title"
        >
          Low transaction fees
        </h3>
        <p>
          Transparent fees designed for teams of all sizes.
        </p>
      </div>
      <div
        class="container"
      >
        <ic-config.svg
          classname="logo"
        />
        <h3
          class="title"
        >
          Get paid fast
        </h3>
        <p>
          Receive your payment within 5-10 days after work is delivered.
        </p>
      </div>
    </div>
  </div>
</div>
`;
